engine/battle/core.asm

```diff
@@ -1007,12 +1007,12 @@ RemoveFaintedPlayerMon:
 	ld b, FLAG_RESET
 	predef FlagActionPredef ; clear gain exp flag for fainted mon
 	ld hl, wEnemyBattleStatus1
-	res 2, [hl]   ; reset "attacking multiple times" flag
+	res ATTACKING_MULTIPLE_TIMES, [hl]
 	ld a, [wLowHealthAlarm]
-	bit 7, a      ; skip sound flag (red bar (?))
+	bit BIT_LOW_HEALTH_ALARM, a
 	jr z, .skipWaitForSound
-	ld a, $ff
-	ld [wLowHealthAlarm], a ;disable low health alarm
+	ld a, (1 << BIT_LOW_HEALTH_ALARM) | $7f
+	ld [wLowHealthAlarm], a
```

Comment:
This $ff is a specific signal value used again on line 3 of audio/low_health_alarm.asm.

Would repeating `(1 << BIT_LOW_HEALTH_ALARM) | $7f` again over there be an improvement?

Reply:
Yes, it should.